src/main/java/teammates/client/scripts/RepairTeamNameInStudentResponseAndCommentAttributes.java

```diff
@@ -0,0 +1,186 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+import javax.jdo.PersistenceManager;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.CommentAttributes;
+import teammates.common.datatransfer.FeedbackResponseAttributes;
+import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.StringHelper;
+import teammates.logic.core.CommentsLogic;
+import teammates.logic.core.FeedbackResponsesLogic;
+import teammates.logic.core.StudentsLogic;
+import teammates.storage.api.StudentsDb;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.Student;
+
+public class RepairTeamNameInStudentResponseAndCommentAttributes extends RemoteApiClient {
```

Comment:
Actually, it is better to have a header comment explaining that the script does. It could be useful when someone wants to reuse/adapt these some day.